documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc

```diff
@@ -83,7 +81,8 @@ spec:
 +
 [source,yaml,options="nowrap",subs=attributes+]
 ----
-...
+spec:
```

Comment:
nitpick: this could be in a separate file since it is shared between "automatic" and "manual" cross-site.